README.md

```diff
@@ -121,6 +152,11 @@ After you're finished please remove all the comments and instructions!
 
 > Describe how a new developer could make use of your project.
 
+<<<<<<< HEAD
+run index.html file
+
+
+=======
```

Comment:
- [ ] Kindly edit the section on line `153`, as it serves as just a guide to you. You're meant to show how your project should be cloned and used.

- [ ] Once again, delete the line `155` and `159` as they are products of a merge